src/platform/FreeBSD.cc

```diff
@@ -31,14 +31,16 @@
 #include "platform/FreeBSD.hh"
 #include "CommandLine.hh"
 #include "UserError.hh"
-
+#include<fcntl.h>
 #include <cassert>
 #include <sstream>
-
```

Comment:
This blank link shouldn't be removed: it separates the C++ headers from the C headers.